lib/parity/environment.rb

```diff
@@ -37,7 +41,9 @@ def backup
     end
 
     def deploy
-      if deploy_to_heroku && run_migration?
+      pending_migrations = run_migration?
+
+      if deploy_to_heroku && pending_migrations
         migrate
       end
```

Comment:
@bernerdschaefer that makes sense. The annoying part is that we have to check for whether we need to migration _before_ we deploy, or else the diff finds nothing to change (this was my mistake on the last version, changing this order). It results in a clunky sequence where we are checking whether we need to migrate, saving that result, then deploying, _then_ running the migrations if needed.